projects/addon-doc/src/services/theme-night.options.ts

```diff
@@ -1,8 +1,15 @@
 import {InjectionToken} from '@angular/core';
+// eslint-disable-next-line @taiga-ui/no-deep-imports
+import {ALWAYS_TRUE_HANDLER} from '@taiga-ui/cdk/constants';
```

Comment:
Why deep import is required?

Reply:
from `addon-doc` we use constant in `cypress` we already use `TUI_DEFAULT_THEME_NAME`, but if you use a short path `cdk`, then `cypress` falls with cyclic dependencies